opentech/apply/stream_forms/files.py

```diff
@@ -16,11 +17,12 @@ def default(self, o):
 
 
 class StreamFieldFile(File):
-    def __init__(self, *args, filename=None, storage=default_storage, **kwargs):
+    def __init__(self, *args, filename=None, storage=default_storage, is_submission=False, **kwargs):
```

Comment:
I'm not entirely happy about this. The stream_form files are meant to be agnostic to the implementation.

There is already a built in method to vary the URL as required. Provide a storage instance instead of the default and that implementation will handle varying the URL.

`submission_storage` from above.